frontend/lib/src/components/elements/Video/Video.tsx

```diff
@@ -49,6 +49,11 @@ export default function Video({
 
     if (videoNode) {
       videoNode.addEventListener("loadedmetadata", setStartTime)
+
```

Comment:
In this case, we will always change the current time to the start time, even if a rerun happens while playing video. 

I think it is a good idea that the new behavior is consistent with `st.audio`

Reply:
Makes sense - I followed `st.audio` pattern this time if you'd like to give it another look!